Makefile

```diff
@@ -41,33 +40,26 @@ endif
 endif
 
 clean:
-	 rm -rf $(DESTINATION) $(ROOT_DIR)/.qemu $(ROOT_DIR)/*.iso $(ROOT_DIR)/*.sha256
+	 rm -rf $(ROOT_DIR)/build $(ROOT_DIR)/.qemu $(ROOT_DIR)/*.iso $(ROOT_DIR)/*.sha256
 
 .PHONY: build
 build:
 	$(LUET) build $(BUILD_ARGS) \
 	--values $(ROOT_DIR)/packages/cOS/values/$(FLAVOR).yaml \
 	--tree=$(TREE) $(PACKAGES) \
-	--destination $(DESTINATION)
+	--destination $(ROOT_DIR)/build
```

Comment:
Maybe we can omit this completely, IIRC it is implied $PWD/build if not specified